src/lib/core/util/rx-operators.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Observable} from 'rxjs/Observable';
+import {Subscription} from 'rxjs/Subscription';
+import {map as _map} from 'rxjs/operator/map';
+import {filter as _filter} from 'rxjs/operator/filter';
+import {reduce as _reduce} from 'rxjs/operator/reduce';
+import {share as _share} from 'rxjs/operator/share';
+import {_finally as __finally} from 'rxjs/operator/finally';
```

Comment:
For `throw` and `finally` can we call them `finallyOperator` and `throwOperator` instead of using an underscore?